data/human/outfits.txt

```diff
@@ -399,6 +409,18 @@ outfit "Local Map"
 	"map" 12
 	description "This data chip contains complete information on the twelve star systems closest to this one: planets, ports, governments, trade prices, available services, etc. You can get all the same information just by exploring those systems yourself, but having a map can save you from making wrong turns if you are trying to travel through new territory to reach a certain system quickly."
 
+outfit "Slave"
+	plural "Slaves"
```

Comment:
```suggestion

```
Unnecessary, the game already automatically add `s` at the end for plural, use `plural` only for special cases where that would be incorrect.